compose.c

```diff
@@ -99,6 +99,7 @@
 char *C_ComposeFormat; ///< Config: printf-like format string for the Compose panel's status bar
 char *C_Ispell; ///< Config: External command to perform spell-checking
 unsigned char C_Postpone; ///< Config: Save messages to the #C_Postponed folder
+bool C_ComposeShowUserHeaders; ///< Config: Whether to display customer user headers
```

Comment:
Ughh I made the same typo here: 'customer'->'custom'. Following my changes to the docs I might just update this (and elsewhere in this file) to 'user-defined headers'. Thoughts/objections @flatcap ?

Reply:
Good idea